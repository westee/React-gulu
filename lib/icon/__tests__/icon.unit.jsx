import React from 'react';
import renderer from 'react-test-renderer';
import Icon from '../../icon/icon';

describe('icon test', ()=>{
    it('是个svg', ()=>{
        const json = renderer.create(<Icon iconName="wechat"/>).toJSON();
        expect(json).toMatchSnapshot()
    })
})