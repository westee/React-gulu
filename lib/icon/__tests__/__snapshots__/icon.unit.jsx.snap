// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon test 是个svg 1`] = `
<svg
  className="fool-icon"
>
  <use
    xlinkHref="#wechat"
  />
</svg>
`;
